cmd/main.go

```diff
@@ -210,6 +213,8 @@ func getJobURL(jobID string) (string, string, error) {
 		}
 		return statusURL, buildID, nil
 	}
+
+	fmt.Printf("Last body:\n%s", string(body))
 	return "", "", fmt.Errorf("status.url not found in response after %d retries", maxAttempts)
```

Comment:
Not sure if there is value to outputting the body or not.  Other consideration is adding backoff and retrying more, forcing the user to quit if they don't want to wait.

Reply:
I think it'd be better to do a back off, with something like ExponentialBackOff here https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait

Doesn't look like this has any delay so we'll frequently hit this if the job doesn't start running immediately

Reply:
There was a sleep in the loop, but I like the backoff / retry more times better.  Made a couple of other small corrections as well.